node_modules_locally.md

```diff
@@ -0,0 +1,8 @@
+1. Create this file: ~/.npmrc
+2. Add this line to the file: 
+- prefix = ${HOME}/.npm-packages
+3. Create npm packages folder:
+- mkdir ~/.npm-packages`
+4. Add this line to .bashrc:
+- PATH=$PATH:$HOME/.npm-packages/bin # Ensure user-global npm binaries can execute.`
+
```

Comment:
@eknelson17  Could you please add details of how you got Sublime configured to work with the local copy?